gc/gc-iceberg-files/build.gradle.kts

```diff
@@ -42,6 +42,9 @@ dependencies {
     exclude("org.eclipse.jetty")
     exclude("org.apache.zookeeper")
   }
+  // Bump the jabx-impl version 2.2.3-1 via hadoop-common to make it work with Java 17+
```

Comment:
The comment seems wrong? In gradle/libs.versions.toml, the version is 2.3.8, not 2.2.3-1.

Reply:
Oh I see, you probably meant "bump from" not "bump to" :-)

Reply:
well, the dependency comes "via" hadoop-common ;)